### PR TITLE
Sat

### PR DESCRIPTION
**Pull request by kent-hervey**
added alex comment to this new sat branch

### CODE REVIEW

**Review by tonylangworthy — APPROVED** [review]

File: src/main/java/com/langworthytech/App.java [hunk 1]
```diff
@@ -107,7 +107,8 @@ public String display(Vehicle vehicle) {
         
         System.out.println("printing make and year" + varMakeYearFilter);
         
-        //Alex is awesome
+        //Alex is awesome very
```
Comment (tonylangworthy) [inline src/main/java/com/langworthytech/App.java]: Please change the comment to "Alex is very awesome"

**Review by tonylangworthy — CHANGES_REQUESTED** [review]
I've left a comment for the changes that need to be made before I can approve the pull request. Thanks!

**Comment by kent-hervey** [comment]
I made changes

**Comment by kent-hervey** [comment]
I made your changes

File: src/main/java/com/langworthytech/App.java (hunk 1 above)
Comment (tonylangworthy) [inline src/main/java/com/langworthytech/App.java]: Please change line 110 to //Alex is very awesome

**Review by tonylangworthy — CHANGES_REQUESTED** [review]

**Review by tonylangworthy — APPROVED** [review]

**Review by tonylangworthy — APPROVED** [review]

**Pull request merged**